coffeebuddy/__init__.py

```diff
@@ -42,8 +52,12 @@ def init_db(app):
     db.init_app(app)
     coffeebuddy.routes.init_routes(app, socketio)
 
-    if app.config['ENV'] in ('development', 'prefilled') or app.testing:
-        db.create_all()
+    if not (app.config['ENV'] == 'sqlite' and not os.path.exists('coffee.db')):
```

Comment:
Not sure if this makes sense? It tries to create the entire database if we're not sqlite or coffee.db exists?